src/styles/getMuiTheme.js

```diff
@@ -31,6 +31,8 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       textColor: palette.alternateTextColor,
       height: spacing.desktopKeylineIncrement,
       titleFontWeight: Typography.fontWeightNormal,
+      paddingLeft: spacing.desktopGutter,
+      paddingRight: spacing.desktopGutter,
```

Comment:
Do we need to separate the `left` property to the `right`? Can't we have a single key?